lib/preprocess.js

```diff
@@ -22,8 +22,10 @@ export class Measurement {
     // Note: providerId is recorded by spark-publish but we don't use it for evaluations yet
     this.providerId = pointerize(m.provider_id)
     this.spark_version = pointerize(m.spark_version)
-    /** @type {import('./typings.js').FraudAssesment} */
-    this.fraudAssessment = null
+    /** @type {import('./typings.js').TaskingEvaluation} */
```

Comment:
Since initial value is set to `null` by default we might should set property type to be optional, maybe something like:

```
/** @type {import('./typings.js').TaskingEvaluation?} */
```

Reply:
Agreed :+1:

This pull request is not making this any worse, so let's leave such improvement out of the scope, please.